dali/kernels/test/warp_test/warp_transform_test.cc

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include "dali/kernels/imgproc/warp/affine.h"
+#include "dali/kernels/imgproc/warp/sphere.h"
+#include "dali/kernels/imgproc/warp/water.h"
```

Comment:
Could you point out, what this file is supposed to test? It has a lot of includes, most of which don't seem to be used

Reply:
At this point it's meant to test that these files actually compile. There hardly exists a way to test these mappings anyway.